### ISSUE TITLE
A function that takes an ID and returns the list of IDs for the group

### ISSUE THREAD

**Issue by MatthewCaseres** [actuarialopensource/pymort#12]
```py
def getGroup(id: int) -> list[int]:
```
This way we can do something like
```py
group = getGroup(3209)
ultimates = jnp.array([MortXML(id).tables[0].Table.numpy for id in group])
```
After we resolve https://github.com/actuarialopensource/pymort/issues/13

**Comment by MatthewCaseres** [comment]
This is implemented in the branch 0.6.0

**Comment by MatthewCaseres** [comment]
still needs to be merged though
